packages/server-wallet/src/handlers/fixtures/update-channel.ts

```diff
@@ -1,16 +1,18 @@
-import {simpleEthAllocation, BN} from '@statechannels/wallet-core';
-
 import {UpdateChannelHandlerParams} from '../update-channel';
-import {alice, bob} from '../../wallet/__test__/fixtures/participants';
 import {fixture} from '../../wallet/__test__/fixtures/utils';
+import {signState} from '../../protocols/actions';
+import {stateVars} from '../../wallet/__test__/fixtures/state-vars';
+
+const lastPostFundTurnNum = 3;
+const channelId = '0x1234';
 
 const defaultVars: UpdateChannelHandlerParams = {
-  channelId: '0x1234',
-  appData: '0x0000000000000000000000000000000000000000000000000000000000000000',
-  outcome: simpleEthAllocation([
-    {destination: alice().destination, amount: BN.from(1)},
-    {destination: bob().destination, amount: BN.from(3)},
-  ]),
+  ...stateVars(),
+  channelId,
 };
 
 export const updateChannelFixture = fixture(defaultVars);
+
+export const signStateFixture = fixture(
+  signState({channelId, ...stateVars({turnNum: lastPostFundTurnNum + 1})})
+);
```

Comment:
Should this be in `protocols/__tests__/fixtures/actions.ts`?

Reply:
Good call: a8861313d